.github/workflows/deployDev.yml

```diff
@@ -18,28 +31,47 @@ jobs:
       REACT_APP_VERSION: 'latest'
       REACT_APP_GOOGLE_ANALYTICS_ID: ${{ secrets.GOOGLE_ANALYTICS_ID }}
       REACT_APP_GRAASP_DOMAIN: ${{ secrets.DEV_GRAASP_DOMAIN }}
+    
+    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      - name: Checkout
+      - name: Check out code
         uses: actions/checkout@v2
 
-      - name: Yarn Set Up
+      - name: Setup node
+        id: setup-node
         uses: actions/setup-node@v2
-      - run: yarn install
-      - run: yarn build
+        with:
+          node-version: '14'
+          check-latest: true
+
+      - name: Create .env.dev file
+        run: |
+          touch .env.dev
+          echo REACT_APP_GRAASP_DOMAIN=dev.graasp >> .env.dev
+          echo REACT_APP_GRAASP_APP_ID=${{ secrets.APP_ID }} >> .env.dev
+          echo REACT_APP_MOCK_API=false >> .env.dev
```

Comment:
Is there a way to input everything in one go? Only if it stays readable of course.

Reply:
Actually I might be wrong but does creating a file necessary? I think setting the variables in `env` is enough. You can check this file https://github.com/graasp/graasp-compose/blob/master/.github/workflows/deployProd.yml
Instead of `yarn build:dev` you can use `yarn dist` (or any command independent of a env file). Locally, we use a file because it's easier than setting by hand the variables in our environment (the files overwrite the local values).